Description: Add a <version> to maven-invoker-plugin or the plugin will not found
Author: Matthias Schmitz <matthias@sigxcpu.org>
Forwarded: not-needed
--- a/animal-sniffer-ant-tasks/pom.xml
+++ b/animal-sniffer-ant-tasks/pom.xml
@@ -94,6 +94,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-invoker-plugin</artifactId>
+	<version>1.3</version>
         <configuration>
           <pomIncludes>
             <pomInclude>*/pom.xml</pomInclude>
--- a/animal-sniffer-enforcer-rule/pom.xml
+++ b/animal-sniffer-enforcer-rule/pom.xml
@@ -97,6 +97,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
+	<version>1.3</version>
         <configuration>
           <setupIncludes>
             <setupInclude>setup-*/pom.xml</setupInclude>
--- a/animal-sniffer-maven-plugin/pom.xml
+++ b/animal-sniffer-maven-plugin/pom.xml
@@ -121,6 +121,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
+	<version>1.3</version>
         <configuration>
           <pomIncludes>
             <pomInclude>*/pom.xml</pomInclude>
